gui/src/components/StoryComponents/StoryDetails.tsx

```diff
@@ -43,6 +43,11 @@ export default function StoryDetails({
     setEditTrue,
   } = useBoardContext();
   const router = useRouter();
+
+  const [issueTitle, setIssueTitle] = useState<string>('');
+  const [issueDescription, setIssueDescription] = useState<string>('');
+  const [actions, setActions] = useState<{ label: string; link: string }[]>([]);
```

Comment:
if you are having more than one prop/keys for a state variable then create an interface for it, export that and use it here